samples/sample_gh_issue_76.eml

```diff
@@ -0,0 +1,4 @@
+From: =?utf-8?q?=20?= <badname@example.com>
```

Comment:
The 'From` header includes my error from #76 (edited the issue today to fix the error there).
Should be one of the following to use a LF or CR instead of a space:
```
From: =?utf-8?q?=0A?= <badname@example.com>
From: =?utf-8?q?=0D?= <badname@example.com>
```

Reply:
Ok, I adapted the sample+test +1